cucumber/authenticators_gcp/features/authn_gcf.feature

```diff
@@ -0,0 +1,70 @@
+Feature: GCP Authenticator - Hosts can authenticate with GCP authenticator
```

Comment:
I think all feature names should also contain GCF or GCE, @eladkug do yo u agree?

Reply:
Need to add global label @GCP before the first feature(it will give us the possibility in the future to run all gcp tests by one command)
Each feature should declare what it testing GCF or GCE.

Reply:
label?

Reply:
This is an example
![image](https://user-images.githubusercontent.com/59792097/93984182-2a96dc00-fd8c-11ea-92c3-7c465eb5f902.png)

Reply:
@eladkug your suggestion is confusing, because the authenticator is GCP, but tokens are from GCE, 
Maybe:
GCP Authenticator - GCE flow, Hosts can authenticate with GCP authenticator

Or

GCP Authenticator - Hosts can authenticate GCP authenticator with GCE token

Reply:
i'm sorry , it was example for the label.
GCP Authenticator - GCE flow, Hosts can authenticate with GCP authenticator --> look good